test/IRGen/abitypes.swift

```diff
@@ -420,19 +435,28 @@ class Foo {
 
   // x86_64-macosx: define hidden swiftcc i1 @_T08abitypes3FooC7negate3S2bF(i1, %T8abitypes3FooC* swiftself) {{.*}} {
   // x86_64-macosx: [[SEL:%[0-9]+]] = load i8*, i8** @"\01L_selector(invert:)", align 8
-  // x86_64-macosx: [[NEG:%[0-9]+]] = call zeroext i1 bitcast (void ()* @objc_msgSend to i1 (%1*, i8*, i1)*)(%1* [[RECEIVER:%[0-9]+]], i8* [[SEL]], i1 zeroext %0)
+  // x86_64-macosx: store i1 %0, i1* %coerced-arg.coerced._value, align 1
+  // x86_64-macosx: [[CAST:%[0-9]+]] = bitcast %TSb* %coerced-arg.coerced to i1*
+  // x86_64-macosx: [[LOAD:%[0-9]+]] = load i1, i1* [[CAST]], align 1
+  // x86_64-macosx: [[NEG:%[0-9]+]] = call zeroext i1 bitcast (void ()* @objc_msgSend to i1 (%1*, i8*, i1)*)(%1* [[RECEIVER:%[0-9]+]], i8* [[SEL]], i1 zeroext [[LOAD]])
   // x86_64-macosx: ret i1 [[NEG]]
   // x86_64-macosx: }
 
   // x86_64-ios: define hidden swiftcc i1 @_T08abitypes3FooC7negate3S2bF(i1, %T8abitypes3FooC* swiftself) {{.*}} {
   // x86_64-ios: [[SEL:%[0-9]+]] = load i8*, i8** @"\01L_selector(invert:)", align 8
-  // x86_64-ios: [[NEG:%[0-9]+]] = call zeroext i1 bitcast (void ()* @objc_msgSend to i1 (%1*, i8*, i1)*)(%1* [[RECEIVER:%[0-9]+]], i8* [[SEL]], i1 zeroext %0)
+  // x86_64-ios: store i1 %0, i1* %coerced-arg.coerced._value, align 1
+  // x86_64-ios: [[CAST:%[0-9]+]] = bitcast %TSb* %coerced-arg.coerced to i1*
```

Comment:
Can these references to ``%coerced-arg.coerced`` reasonably be a ``[[VARIABLE]]`` reference instead?

Reply:
Not sure about the x86_64 macosx case, but the others ones are regular enough to do.